farcy/__init__.py

```diff
@@ -46,6 +46,8 @@
 VERSION_STR = 'farcy v{0}'.format(__version__)
 MD_VERSION_STR = ('[{0}](https://github.com/appfolio/farcy)'
                   .format(VERSION_STR))
+PR_ISSUE_COMMENT_FORMAT = '_{}_ {{0}}'.format(MD_VERSION_STR)
```

Comment:
Can we use the `{0}` format here? I know we're not supporting py 2.6, though I think it is a bit simpler to read.